corehq/apps/app_manager/models.py

```diff
@@ -1457,7 +1457,8 @@ def audio_by_language(self, lang, strict=False):
     def custom_icon_form_and_text_by_language(self, lang):
         custom_icon = self.custom_icon
         if custom_icon:
-            return custom_icon.form, custom_icon.text[lang]
+            custom_icon_text = custom_icon.text.get(lang) or custom_icon.text.get(self.get_app().default_language)
```

Comment:
could be more concise: `custom_icon.text.get(lang, self.get_app().default_language)`

Reply:
I think you'd want this:
```python
custom_icon.text.get(lang, custom_icon.text.get(self.get_app().default_language))
```

Reply:
oops, i was sloppy - thanks Ethan :)

Reply:
Actually did the opposite to reduce nesting.